### PR TITLE
Reconcilation command line 

### LINKED ISSUE #586: `cyctl` reconcile module command
**Is your feature request related to a problem? Please describe.**
Implement a `cyctl` subcommand to trigger module reconciliation. The subcommand does the same thing as the reconcile button on the module details page.

Screenshot 2024-09-23 at 15 57 18

The subcommand should look like:
```
cyctl reconcile <module name>
```

**Describe the solution you'd like**
You can trigger the reconciliation by fetching the module, adding the `cyclops/reconciled-at` with the current timestamp and updating the Module using a kubernetes client.

You can check the implementation of the same reconciliation triggering in `cyclops-ctrl` [here](https://github.com/cyclops-ui/cyclops/blob/cdc16dc449845c920b0e77dab6a8139c9b627cfe/cyclops-ctrl/internal/controller/modules.go#L350).

### PR DESCRIPTION
**Pull request by Abiji-2020**
closes #586 

## 📑 Description

created a function in Cobra and added in the root command running `cyctl reconcile <module name>` will reconcile the specified module;

## ✅ Checks
- [x] I have tested my code (provide screenshots or screen recordings of a working solution)
- [x] I have performed a self-review of my code

### CODE REVIEW

File: cyctl/cmd/reconcile.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+var (
+	reconcileExample = `# Reconcile the Module 
+	cyctl reconcile <modulename>`
+)
+
+var reconcileCMD = &cobra.Command{
+	Use:     "reconcile",
+	Short:   "Will reconcile the Module and update the current TimeStamp",
+	Long:    "Will reconcile the Module and update the current TimeStamp",
```
Comment (petar-cvit) [inline cyctl/cmd/reconcile.go:47]:
```suggestion
	Short:   "Trigger module reconciliation",
	Long:    "Trigger module reconciliation",
```

File: cyctl/cmd/reconcile.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		APIVersion: "cyclops-ui.com/v1alpha1",
+		Kind:       "Module",
+	}
+
+	_, err = clientset.Modules("cyclops").Update(module)
+	if err != nil {
+		fmt.Println("failed to update module: ", err)
+		return
+	}
+	fmt.Printf("successfully reconciled %v", moduleName)
```
Comment (petar-cvit) [inline cyctl/cmd/reconcile.go:35]:
```suggestion
	fmt.Printf("successfully triggered reconciliation for module: %v", moduleName)
```

File: cyctl/cmd/reconcile.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+}
+
+var (
+	reconcileExample = `# Reconcile the Module
```
Comment (petar-cvit) [inline cyctl/cmd/reconcile.go:40]:
```suggestion
	reconcileExample = `# Reconcile a Module 
```

**Review by petar-cvit — CHANGES_REQUESTED** [review]
@Abiji-2020 thanks for the PR! Let a couple of comments on cli output, but looks good

File: cyctl/cmd/reconcile.go (hunk 3 above)
Comment (Abiji-2020) [reply cyctl/cmd/reconcile.go]: Finished the requested changes

**Comment by Abiji-2020** [comment]
@petar-cvit  The requested changes on the outputs of the cli have been updated

**Review by petar-cvit — APPROVED** [review]
Thanks @Abiji-2020 🧡

**Pull request merged**